forge/housekeeper/tasks/expireTokens.js

```diff
@@ -1,9 +1,13 @@
 const { Op } = require('sequelize')
 
+const randomInt = (min, max) => { return min + Math.floor(Math.random() * (max - min)) }
+
 module.exports = {
     name: 'expireTokens',
     startup: true,
-    schedule: '@daily',
+    // Pick a random hour/minute for this task to run at. If the application is
+    // horizontal scaled, this will avoid two instances running at the same time
+    schedule: `${randomInt(0, 60)} ${randomInt(0, 24)} * * *`,
```

Comment:
Only apply if first suggestion is accepted.

```suggestion
    schedule: `${randomInt(0, 59)} ${randomInt(0, 23)} * * *`,
```

Reply:
Resolved by https://github.com/FlowFuse/flowfuse/pull/3327/commits/9bb33fa445c23daa24226ecce5df43bb1054d309